package.json

```diff
@@ -26,5 +26,8 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "dependencies": {
+    "@heroku-cli/command": "^8.2.2"
```

Comment:
this shouldn't be included

Reply:
@jdxcode yeah, I just noticed that.  Should be gone now.